test/commands/lwc/test/create.test.ts

```diff
@@ -34,7 +34,7 @@ describe('force:lightning:lwc:test:create', () => {
       stubMethod($$.SANDBOX, fs, 'writeFileSync');
     })
     .stdout()
-    .withProject()
+    // .withProject()
```

Comment:
I commented this out because there's a bug in the new version where the project is already stubbed and that causes errors.  I'll work on the bug fix instead so I can leave these in since updating this plugin is no longer urgent.